src/ray/stats/metric_defs.cc

```diff
@@ -196,6 +196,18 @@ DEFINE_stats(gcs_storage_operation_count,
              (),
              ray::stats::COUNT);
 
+/// Object store
+// TODO:
+DEFINE_stats(object_store_memory,
+             "Object store memory by various sub-kinds on this node",
+             /// Location:
+             ///    - InMemory: currently in shared memory(e.g. /dev/shm)
+             ///    - Spilled: spilled to external storage
```

Comment:
I think it's important we include spilling this in this PR, since it impacts the design of the other recording points. Can we actually punt fallback allocated to another PR (just treat it as InMemory?)